src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,13 +1,37 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+    private String excavatorChassisType;
+    private String attachments;
+    private String excavatorType;
```

Comment:
The fields `excavatorChassisType`, `attachments`, and `excavatorType` should have private access modifiers to encapsulate the properties of the Excavator class. Direct access to class fields is discouraged and they should be accessed through getters and setters.